test/extension.test.ts

```diff
@@ -26,9 +27,20 @@ import {OutputConsumer} from '../src/proc';
 import * as state from '../src/state';
 import * as util from '../src/util';
 
-
 const here = __dirname;
 
+function clearExistingKitConfigurationFile() {
+  fs.writeFile( path.join(dirs.dataDir, 'cmake-kits.json'), "[]");
+}
+
+async function getExtension() {
+  const cmt = vscode.extensions.getExtension<CMakeTools>('vector-of-bool.cmake-tools');
+  if (!cmt) {
+    return Promise.reject("Extension doesn't exist");
```

Comment:
Didn't @vector-of-bool mention something along the lines of using exceptions for rejecting promises?

Reply:
Done